packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -146,9 +147,19 @@ export class AtomicSegmentedFacet
   }
 
   public render() {
+    if (
+      this.searchStatus.state.hasError ||
+      !this.facetState.values.length ||
+      !this.facet.state.enabled
+    ) {
+      return <Hidden></Hidden>;
+    }
+
     return (
       <div class="flex">
-        <b class="inline-block my-3 mr-2">{this.label}</b>
+        <b class="inline-block my-3 mr-2">
+          {this.label !== 'no-label' ? this.label.concat(':') : ''}
```

Comment:
Instead of having a default value of `no-label` at line 61, I think we should make label optional like so:
```ts
  @Prop({reflect: true}) public label?: string;
```

That way, here, you can instead compare `this.label` against `undefined` like so:
```suggestion
          {this.label !== undefined ? this.label.concat(':') : ''}
```

The reason normal facets had a default value of `no-label` was because it's the localization string for the string "No label"/"Sans titre".

Reply:
Sure, will fix !

Reply:
For components that require a label, should I pass 'no-label' or alter those components to make a label optional

Reply:
alter those components